.env

```diff
@@ -5,7 +5,7 @@ DATASTORE_READONLY_PASSWORD=datastore
 # Basic
 CKAN_SITE_ID=default
 PORT=5000
-CKAN_SITE_URL=http://ckan:5000
+CKAN_SITE_URL=http://ckan:$PORT
```

Comment:
I don't know that this will work the way you expect. I'm not sure how this is pulled in via Docker, but I don't know if each line is handled individually or it is "sourced". I don't know if you can chain variables in the way you do here...

Reply:
Yeah, that's probably why the tests are failing.  But either way, this was not ready yet haha... but thanks for looking at it still 😅 (You too haha @btylerburton)

Reply:
You were dead on the money @jbrown-xentity, you fixed it 😄 

It's still a work in progress though haha..